nuxt.config.js

```diff
@@ -21,12 +21,17 @@ export default {
   meta: {
     nativeUI: true,
     lang: 'de',
+    name: 'SplusEins',
+    author: 'SplusEins-Team',
+    description: 'Stundenplan Plus Mehr für Studenten der Ostfalia.'
```

Comment:
Das steht zum Beispiel in den Google-Suchergebnissen und wenn man den Link über WhatsApp oder Discord teilt.